ui/src/components/Settings/About/index.tsx

```diff
@@ -0,0 +1,322 @@
+  useEffect(() => {
+    GetApiHandler('/rules/community').then((resp) => {
+      if (resp) {
+        let totalCount = 0
+        if (Array.isArray(resp)) {
+          totalCount = resp.length
+        }
+        setCommunityCount(totalCount)
+      }
+    })
+  })
+  return (
+    <div className="h-full w-full">
+      <div className="mt-6 rounded-md border border-amber-600 bg-amber-500 bg-opacity-20 p-4 backdrop-blur">
+        <div className="flex">
+          <div className="flex-shrink-0">
+            <InformationCircleIcon className="h-5 w-5 text-gray-100" />
+          </div>
+          <div className="ml-3 flex-1 md:flex md:justify-between">
+            <p className="text-sm leading-5 text-gray-100">
+              This is BETA software. Features may be broken and/or unstable.
+              Please report any issues on GitHub!
+            </p>
+            <p className="mt-3 text-sm leading-5 md:ml-6 md:mt-0">
+              <a
+                href="http://github.com/jorenn92/maintainerr"
+                className="whitespace-nowrap font-medium text-gray-100 transition duration-150 ease-in-out hover:text-white"
+                target="_blank"
+                rel="noreferrer"
+              >
+                GitHub &rarr;
+              </a>
+            </p>
+          </div>
+        </div>
+      </div>
+      {/* Maintainerr Portion */}
+      <div className="section mb-2 h-full w-full">
+        <h3 className="heading">About Maintainerr</h3>
+      </div>
+      <hr className="my-2 h-px border-0 bg-gray-200 dark:bg-gray-700"></hr>
+      <div className="section my-2">
+        <div className="form-row my-2">
+          <label htmlFor="name" className="text-label">
+            Version
+          </label>
+          <div className="form-input">
+            <div className="form-input-field">
+              <span className="">
+                <code>{commitTag === 'local' ? 'local' : <>{version}</>}</code>
+              </span>
+            </div>
+          </div>
+        </div>
+        <hr className="my-2 h-px border-0 bg-gray-200 dark:bg-gray-700"></hr>
+        <div className="form-row my-2">
+          <label htmlFor="name" className="text-label">
+            Container Config Path
+          </label>
+          <div className="form-input">
+            <div className="form-input-field">
+              <span className="">
+                <code>/opt/data</code>
+              </span>
+            </div>
+          </div>
+        </div>
+        <hr className="my-2 h-px border-0 bg-gray-200 dark:bg-gray-700"></hr>
+        <div className="form-row my-2">
+          <label htmlFor="name" className="text-label">
+            Time Zone
+          </label>
+          <div className="form-input">
+            <div className="form-input-field">
+              <span className="">
+                <code>{getBrowserTimezone()}</code>
```

Comment:
I'm not sure that the time zone being here is useful. The server might be using a different timezone to what the browser is using.

Reply:
I agree. I wanted some more data in there when I initially wrote it, but didn't know how to get what TZ the container was set to. Ideally, I would get the information from the container's environment variables. I would like to use that for the host data volume location as well. Do you know how we can get docker environment information into the app?

Reply:
For the timezone you can return it from the API. This will return the IANA time zone: `Intl.DateTimeFormat().resolvedOptions().timeZone`

There is no environment variable for the data volume, it's always `/opt/data` so hard coding it like you have is fine.

Reply:
I would like to pull the TZ information from whatever environment variable they have set in their compose. 
It is currently coded like you stated. 
```
  const getBrowserTimezone = (): string => {
    return Intl.DateTimeFormat().resolvedOptions().timeZone
  }
```
As for the /opt/data... I was thinking of showing what host dir they have /opt/data mapped to. Probably a little much. Should just keep it simple.

Reply:
Your code is being executed on the client so it will show the browser's time zone. Run what you've got on the server and it will return the time zone that Node is using, which it will get from the environment.

Reply:
> Your code is being executed on the client so it will show the browser's time zone. Run what you've got on the server and it will return the time zone that Node is using, which it will get from the environment.

Makes sense.